frontend/src/components/SeriesItem.tsx

```diff
@@ -0,0 +1,29 @@
+import {Series} from "../models/Series";
+import MonetaryValue from "./MonetaryValue";
+import FormatDate from "./FormatDate";
+
+interface ISeriesItemProps {
+    series: Series
+    deleteSeries: (seriesId?: string) => void
+}
+
+export default function SeriesItem({series, deleteSeries}: ISeriesItemProps) {
+
+    return (
+        <div className={"seriesItemCard"}>
+            <input type="button" onClick={() => deleteSeries(series.seriesId)} value={"Delete"}/>
+            <div className={"seriesItem"}>
+                <h2>Start</h2>
+                <h3>{series.startDate ? <FormatDate date={series.startDate}/> : "Infinite"}</h3>
+                <h2>End</h2>
+                <h3>{series.endDate ? <FormatDate date={series.endDate}/> : "Infinite"}</h3>
+                <h2>Scheduled Day</h2>
+                <h3>{series.scheduledDate}</h3>
```

Comment:
Coach Jan commented a similar part in my last PR that this type of heading usage breaks the heading semantics.